### PR TITLE
Adding remaining HMRC unaudited company validations

### PR DESCRIPTION
**Pull request by allenbross-wf**
#### Description of change

Added validations: 
Unaudited Micro-LLP, 
Unaudited Company Abridged Accounts, 
Unaudited LLP Abridged Accounts, 
Unaudited Company Abbreviated Accounts, 
Unaudited LLP Abbreviated Accounts, 
Unaudited Company Group Accounts, 
Unaudited LLP Group Accounts, 
Unaudited Small Company Full Accounts, 
Unaudited LLP Full Accounts

#### Steps to Test

CI.  Conformance suite has been updated for each validation.

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/validate/HMRC/ValidateHmrc.py [hunk 1]
```diff
@@ -367,30 +434,120 @@ def validate(self) -> None:
                 else:
```
Comment (derekgengenbacher-wf) [inline arelle/plugin/validate/HMRC/ValidateHmrc.py:434]: The lines 428 and 429 are the same.

File: arelle/plugin/validate/HMRC/ValidateHmrc.py [hunk 2]
```diff
@@ -367,30 +434,120 @@ def validate(self) -> None:
...
+        result = self._evaluateCode(CO_SM_CO)
+        if not result.success:
+            self._errorOnMissingFactText(CO_SM_CO, result)
 
-        result = self._evaluateCode(CO_MICRO)
+    def validateUnauditedCompanyAbridgedAccounts(self) -> None:
+        """
+        Checks conditions applicable to unaudited company abridged accounts:
+        Co.Sec777, Co.AuditNR, Co.DirResp, Co.SmCo, Co.Abrid
+        :return:
```
Comment (derekgengenbacher-wf) [inline arelle/plugin/validate/HMRC/ValidateHmrc.py:492]: I would either use this and document the return. Or remove it. Up to you.

File: arelle/plugin/validate/HMRC/ValidateHmrc.py (hunk 1 above)
Comment (allenbross-wf) [reply arelle/plugin/validate/HMRC/ValidateHmrc.py:434]: Updated and updated testcase invalid-lp-sec477.xbrl to verify

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Testing locally succeeded.

**Pull request merged**